WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -6,14 +6,35 @@
 import androidx.annotation.Nullable;
 
 import org.wordpress.android.fluxc.model.SiteModel;
+import org.wordpress.android.fluxc.store.AccountStore;
 import org.wordpress.android.fluxc.store.SiteStore;
 import org.wordpress.android.ui.plans.PlansConstants;
+import org.wordpress.android.ui.prefs.AppPrefs;
 import org.wordpress.android.util.helpers.Version;
 
 import java.util.ArrayList;
 import java.util.List;
 
 public class SiteUtils {
+    public static final int MIN_WPCOM_USER_ID_TO_DEFAULT_GB = 149287441;
+
+    public static boolean isBlockEditorDefaultForNewPost(SiteModel site, AccountStore accountStore) {
```

Comment:
It seems that the `accountStore` can be `null` and is even marked as nullable [here](https://github.com/wordpress-mobile/WordPress-Android/blob/673115d1ece0620acd0e181721584d3f72a3742f/WordPress/src/main/java/org/wordpress/android/ui/accounts/HelpActivity.kt#L171) and [here](https://github.com/wordpress-mobile/WordPress-Android/blob/5573e47b1c97da0834c66b6772923e7ffc015a17/WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java#L629). My impression is that it shouldn't be nullable at all since the Stores are supposed to be available.

Reply:
Apparently fixed with d2f8442d2c80ecff47d95fb0b8ba4fe9caeaef68.